Assets/Scripts/UI/Speech/SpeechZone.cs

```diff
@@ -101,8 +103,12 @@ public class SpeechZone : MonoBehaviour {
 			messageText.text = "";
 			messageText.gameObject.SetActive(false);
 		}
-
 		isShowing = false;
+
+		if (playerController != null) {
+			playerController.enabled = true;
+			playerAnimator.enabled = true;
+		}
```

Comment:
Why does the speechzone control when to give the player control again? Shouldn't this be handled in the haltplayer script?